pkg/cli/mirror/options.go

```diff
@@ -73,6 +74,7 @@ func (o *MirrorOptions) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&o.OCIRegistriesConfig, "oci-registries-config", o.OCIRegistriesConfig, "Registries config file location (it works only with local oci catalogs)")
 	fs.BoolVar(&o.OCIInsecureSignaturePolicy, "oci-insecure-signature-policy", o.OCIInsecureSignaturePolicy, "If set, OCI catalog push will not try to push signatures")
 	fs.BoolVar(&o.SkipPruning, "skip-pruning", o.SkipPruning, "If set, will disable pruning globally")
+	fs.BoolVar(&o.SkipPruningMissingManifests, "skip-pruning-missing-manifests", o.SkipPruningMissingManifests, "If set, pruning will not fail if it's unable to find the manifest")
```

Comment:
I would say it should be the default behavior. Why would one want the pruner to fail when it gets 404 on a delete request?

Reply:
I would agree with this - @sherine-k @lmzuccarelli any objections?

Reply:
@bcaton85 - no objections

Reply:
okay let me update

Reply:
updated